### ISSUE TITLE
The type  of the orderinput() return values is been changed

### ISSUE THREAD

**Issue by Yinlingzhu** [Yang-Tang/shinyjqui#84]
The shinyjqui package update to the latest version causes the orderinput() return value from vector to data frame
now: 
  id                text
1 ui1   Protein accession
2 ui1 Protein description
3 ui1           Gene name

old:
Protein accession Protein description Gene name

**Comment by Yang-Tang** [comment]
Yes, you are right. From v0.4.0, there is a breaking change of `orderinput()` to make it behave more like a shiny's native input function. To get a vector of return value, you should now use `input$inputId` instead of `input$inputId_order`.
